conda-forge.yml

```diff
@@ -4,7 +4,7 @@ build_platform:
   linux_aarch64: linux_64
   linux_ppc64le: linux_64
   osx_arm64: osx_64
-build_with_mambabuild: false
+conda_build_tool: 'conda-build+conda-libmamba-solver'
```

Comment:
I turned on `libmamba-solver` here as I didn't see any reason not to. Though if this works is there any reason to not use `mambabuild`?

Reply:
`mambabuild` (and `boa`) is effectively dead as far as I know. Now the libmamba solver can be used with conda it doesn't make sense any more.

The work on a new recipe format that was in boa is now happening in rattler-build.

Reply:
Thanks @chrisburr. That's useful knowledge!

Also thanks for the fast review and merge.